en/docs/references/adaptive-authentication-js-api-reference.md

```diff
@@ -100,7 +100,16 @@ The API can be called in either of the following ways:
         }
     });
     ```
-      
+    ``` java
+    executeStep(1, {
+        stepOptions: {
+            forceAuth: 'true',
+            subjectIdentifier: 'true',
+            subjectAttributeStep: 'true'
+        }
+    });
```

Comment:
```suggestion
    }, {} );
```

We should always pass a callback as the third option to this work. That can be empty.